sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/CosmosSourceConnector.java

```diff
@@ -55,11 +56,14 @@
  */
 public final class CosmosSourceConnector extends SourceConnector implements AutoCloseable {
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosSourceConnector.class);
+    private static final String CONNECTOR_NAME = "name";
```

Comment:
what is the default value here? Just the string `name`?

Reply:
the connector name should actually be forced from kafka side, during my testing, if I do not provide a name, an error will be thrown from kafka side, so supposedly we should not never get an empty name